packages/cozy-client/src/store/queries.js

```diff
@@ -79,80 +82,107 @@ const query = (state = queryInitialState, action) => {
   }
 }
 
-const queryContains = (query, id) => {
-  return query.data.indexOf(id) > -1
-}
-
-const findQueriesContaining = (queries, id) => {
-  return fromPairs(
-    Object.entries(queries).filter(([queryId, query]) =>
-      queryContains(query, id)
-    )
-  )
+const filterForQuery = query => {
+  const qdoctype = query.definition.doctype
+  const selectorFilterFn = query.definition.selector
+    ? selectorFilter(query.definition.selector)
+    : null
+  return datum => {
```

Comment:
I don't want to nitpick about variable names, but what does "datum" mean in this context ?

Reply:
Just one doc. d3 uses the same convention. One piece of data is a datum. No problem about nitpicking about variable names.